sirepo/job_driver/sbatch.py

```diff
@@ -102,32 +104,76 @@ def _agent_env(self):
         )
 
     async def _do_agent_start(self, msg):
+        log_file = 'job_agent.log'
+        agent_start_dir = self._srdb_root
+        script = f'''#!/bin/bash
+{self._agent_start_dev()}
+set -e
+mkdir -p '{agent_start_dir}'
+cd '{self._srdb_root}'
+{self._agent_env()}
+(/usr/bin/env; setsid {cfg.sirepo_cmd} job_agent start_sbatch) >& {log_file} &
+disown
+'''
+
+        def write_to_log(stdout, stderr, filename):
+            p = pkio.py_path(msg.userDir).join('log')
+            if not p.exists():
+                pkio.mkdir_parent(p)
+            pkjson.dump_pretty(
+                PKDict(stdout=stdout, stderr=stderr),
+                p.join(f'{int(time.time())}-{filename}.log'),
```

Comment:
`datetime.datetime.now().strftime('%Y%m%d%H%M%S')`

This should be encapsulated, but a little copying at this point...